junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvSource.java

```diff
@@ -22,7 +22,7 @@
 
 /**
  * {@code @CsvSource} is an {@link ArgumentsSource} which reads
- * comma-separated values (CSV) from one or more supplied
+ * delimiter-separated (by default with coma) values (CSV) from one or more supplied
```

Comment:
I think we should keep this as-is, but add a paragraph below this one that explains that the delimiter can be customized.